src/screens/Podcast/Podcast.tsx

```diff
@@ -43,6 +43,8 @@ const Podcast: React.FC = () => {
   const route = useRoute<PodcastScreenRouteProp>()
   const navigation = useNavigation<PodcastScreenNavigationProp>()
 
+  const DEFAULT_PODCAST_BACKGROUND = 'http://hexcolor16.com/eeeeee-200x200.png'
```

Comment:
Using third party resources for default image not good. We create "DEFAULT_IMAGE_BASE64" special and agreed it with Zhenya.

Reply:
Ok, will fix it